diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -62,6 +62,8 @@ class DiktatRuleSetProvider(private val diktatConfigFile: String = "diktat-analy
                 ::PackageNaming,
                 ::IdentifierNaming,
                 // code structure
+                ::UselessSupertype,
+                ::LocalVariablesRule,
```

Comment:
Did you move `LocalVariablesRule` on purpose?

Reply:
No, it was my a mistake during the merge

Reply:
Please move `LocalVariablesRule` back if it's unintended move, the rest seems fine.